README.md

```diff
@@ -0,0 +1,31 @@
+# Octavia Provider Driver for F5 BigIP LTM devices
+
+This is the [Octavia](https://github.com/sapcc/octavia) provider driver for F5 BigIP LTM appliances.
+It communicates with BigIP devices via the declarative [AS3 API](https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/).
+
+Note: What we call 'worker' is called 'provider agent' in the [Octavia Provider Driver Development Guide](https://docs.openstack.org/octavia/latest/contributor/guides/providers.html).
+
+# Entrypoints
+All scripts are called with the following parameters: `--debug --config-file ~/.config/octavia.conf`
+All scripts are called with the following environment variables: `PYTHONUNBUFFERED=1;PYTHONWARNINGS=ignore:Unverified HTTPS request`
```

Comment:
I would remove this section, the config options are are the same as with upstream octavia and the environment variable is also generic and just used by me (pythonunbuffered is pycharm default and pythonwarnings override is used to silence pythonwarnings about certificate problems - let's not suggest that security-wise).

Probably it would make more sense to add a list of the f5 specific config-options from common/config.py

Reply:
Instead of adding a whole list I'll just point to `octavia_f5/common/config.py`, since it doesn't make sense to maintain the options in two different places.

Reply:
ok